src/captcha.ts

```diff
@@ -194,6 +190,11 @@ export class WidgetInstance {
       return;
     }
 
+    if (this.hasBeenStarted) {
+      console.error("Can not start FriendlyCaptcha widget which has already been started");
```

Comment:
```suggestion
      console.warn("Can not start FriendlyCaptcha widget which has already been started");
```

I think warn (or maybe even info or debug) is enough here, we've made it idempotent more or less with this change so it's not the end of the world if someone's implementation `start`s the widget another time